assets/js/components/modal.js

```diff
@@ -19,28 +19,23 @@
 /**
  * WordPress dependencies
  */
-import { Component, createPortal } from '@wordpress/element';
+import { createPortal, useEffect, useState } from '@wordpress/element';
```

Comment:
I think this file was accidentally kept. It should be removed in favor of `Modal.js`.